src/summary/documentation/dimensions-card-level-two.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styles from "./dimensions-card-level-two.css";
+import SummaryCard from "../cards/summary-card.component";
+import { getDimensions } from "./dimensions-card.resource";
+import { match } from "react-router";
+
+function DimensionsCardLevelTwo(props: DimensionsCardLevelTwoProps) {
+  const [dimensions, setDimensions] = React.useState([]);
+
+  React.useEffect(() => {
+    const sub = getDimensions(props.currentPatient.id).subscribe(dimensions =>
+      setDimensions(dimensions)
+    );
+    return () => sub.unsubscribe();
+  }, []);
+
+  return (
+    <div style={{ display: "flex" }}>
+      <SummaryCard
+        name="Height & Weight"
+        match={props.match}
+        styles={{ flex: 1, margin: ".5rem" }}
+      >
+        <table className={styles.table}>
+          <thead>
+            <tr className={styles.tableRow}>
+              <th
+                className={`${styles.tableHeader} ${styles.tableDates}`}
+                style={{ textAlign: "start" }}
+              ></th>
+              <th className={styles.tableHeader}>Weight (kg)</th>
+              <th className={styles.tableHeader}>Height (cm)</th>
+              <th className={styles.tableHeader}>
+                BMI (kg/m<sup>2</sup>)
+              </th>
+              <th></th>
+            </tr>
+          </thead>
+          <tbody>
+            {dimensions.map(dimension => (
+              <tr key={dimension.id} className={styles.tableRow}>
+                <td className={styles.tableData} style={{ textAlign: "start" }}>
+                  <span style={{ fontWeight: 500 }}>
+                    {dimension.date.split(" ")[0]}
```

Comment:
Instead of splitting strings and stuff, why don't we just do `dayjs(dimension.date).format()` multiple times. We can do it for the bolded span and for the non-bolded span.

Reply:
`dimension.date` is not necessarily a date, but a formatted date according to the designs, it could look something like this `Today 06:45AM` so it's a string.

Reply:
Fair enough